lms/templates/sysadmin_dashboard.html

```diff
@@ -189,7 +189,7 @@ <h2 class="instructor-nav">
 %endif
 
 %if modeflag.get('task_queue'):
-  <h3>${_('Kill Queuing Tasks')}</h3><br/>
+  <h3>${_('Kill Progess or Queuing Task')}</h3><br/>
```

Comment:
> "Set task state to failure" would be more clear.

I disagree; that's leaking implementation details.

Beyond the verbosity, if edX later changes the underlying implementation [1],
maybe they change the name of the failed/canceled state or some other
details, then this becomes a maintenance burden. Now we have to keep
user-facing documentation in-sync with details of the technical
implementation. Then someone goes to "set task state to failure" but
can't find that option because now the text has been changed again.
Spoiler Alert: edX already has plans to rework this.

That the way to "kill" a task here happens to be a specific database
modification shouldn't be of immediate concern to the user.
I agree that "kill" could possibly be misleading without context; it's
admittedly not obvious that it doesn't halt a running process.
But as it were, "kill task" is the colloquialism that our team has
historically used.

Here are some snippets from the last email that Monica sent regarding
this issue:

> Also, since grade report activities are not displaying in the list of
> Task Queues, there isn't any way for us to kill this task.

and

> Once this failed task is killed [...]

Seeing as this is how we've always described this action, my preference
is to not introduce new vernacular at this point.

[1] https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/edx-code/17DA6dt2Dmc/cdoFMMinBAAJ